jlm/llvm/opt/alias-analyses/Andersen.cpp

```diff
@@ -0,0 +1,702 @@
+/*
+ * Copyright 2023 Håvard Krogstie <krogstie.havard@gmail.com>
+ * See COPYING for terms of redistribution.
+ */
+
+#include <jlm/llvm/opt/alias-analyses/Andersen.hpp>
+#include <jlm/llvm/opt/alias-analyses/PointsToGraph.hpp>
+#include <jlm/rvsdg/node.hpp>
+#include <jlm/rvsdg/traverser.hpp>
+#include <jlm/util/Statistics.hpp>
+
+#include <typeinfo>
+
+namespace jlm::llvm::aa
+{
+
+void
+Andersen::AnalyzeSimpleNode(const rvsdg::simple_node & node)
+{
+  const auto & op = node.operation();
+  const auto & type = typeid(op);
+
+  if (type == typeid(alloca_op))
```

Comment:
Could you use [is()](https://github.com/phate/jlm/blob/f2e1a788b57d40b0438223e195d9afd776b1dbd8/jlm/rvsdg/node.hpp#L903C1-L911C2) here? It would be more readable.